data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -48,23 +48,23 @@
 {% endif %}
 
 {% if has_start_date %}
-WHERE (COALESCE({{end_date}}, EXTRACT(DATE FROM {{end_datetime}}),
-    {{start_date}}, EXTRACT(DATE FROM {{start_datetime}})) >= d.deactivated_date
```

Comment:
Please update the column name from `deactivated_date` to `deactivated_datetime`, as @molly-giddings's DC-1795 will be also updating the column name.

Reply:
@hiro-mishima Thanks for cross-referencing these two tickets. Do you think it would be best to hold off on the final work for this until DC-1795 is merged?